src/axolotl/core/trainer_builder.py

```diff
@@ -1532,6 +1539,18 @@ def build_training_arguments(self, total_num_steps):
             if self.cfg.max_prompt_len:
                 training_args_kwargs["max_prompt_length"] = self.cfg.max_prompt_len
 
+        if self.cfg.rl == "kto":
+            training_args_cls = KTOConfig
+
+            training_args_kwargs["beta"] = self.cfg.dpo_beta or 0.1
```

Comment:
IMO `dpo_beta` is slightly confusing here (because this isn't dpo), but it pretty much serves the same purpose. should I introduce a `kto_beta` config value?

Reply:
let's change it to `rl_beta`. We can add the dpo_beta to [DeprecatedParameters](https://github.com/OpenAccess-AI-Collective/axolotl/blob/main/src/axolotl/utils/config/models/input/v0_4_1/__init__.py#L21) and if it is set, we can map it to the new `rl_beta` similar to https://github.com/OpenAccess-AI-Collective/axolotl/blob/main/src/axolotl/utils/config/models/input/v0_4_1/__init__.py#L863-L868

Reply:
I went ahead and changed this and updated the validation.